.github/workflows/root-ci-config/buildconfig/fedora40.txt

```diff
@@ -0,0 +1,4 @@
+builtin_nlohmannjson=On
+builtin_vdt=On
+pythia8=Off
+tmva-pymva=Off
```

Comment:
```suggestion
```

As of #15439, this is not done done anymore for Fedora 39. Probably for Fedora 40, we shouldn't do this customization either